android-client-sdk/src/main/java/com/devcycle/sdk/android/model/DVCUser.kt

```diff
@@ -2,7 +2,6 @@ package com.devcycle.sdk.android.model
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
-import java.util.*
```

Comment:
I assume this was just un-used?

Reply:
yeah, missed cleanup in previous releases so doing it now